src/main/webapp/app/exam/participate/exam-participation.service.ts

```diff
@@ -113,16 +113,15 @@ export class ExamParticipationService {
      * @param courseId the id of the course the exam is created in
      * @param examId the id of the exam
      * @param userId the id of the student if the current caller is an instructor, the grade info for current user's exam will be retrieved if this argument is empty
-     * @param isTestRun
+     * @param studentExamId the id of the student exam
      */
-    public loadStudentExamGradeInfoForSummary(courseId: number, examId: number, userId?: number, isTestRun?: boolean): Observable<StudentExamWithGradeDTO> {
+    public loadStudentExamGradeInfoForSummary(courseId: number, examId: number, studentExamId: number, userId?: number): Observable<StudentExamWithGradeDTO> {
         let params = new HttpParams();
         if (userId) {
             params = params.set('userId', userId.toString());
         }
-        params = params.append('isTestRun', !!isTestRun);
 
-        const url = this.getResourceURL(courseId, examId) + '/student-exams/grade-summary';
+        const url = this.getResourceURL(courseId, examId) + '/student-exams/' + studentExamId + '/grade-summary';
```

Comment:
I agree with code rabbit here:
```suggestion
        const url = `${this.getResourceURL(courseId, examId)}/student-exams/${studentExamId}/grade-summary`;
```